examples/custom-assessment/job.yml

```diff
@@ -0,0 +1,27 @@
+$schema: https://azuremlschemas.azureedge.net/latest/commandJob.schema.json
+code: src
+command: >-
+  python main.py
+  --checkpoint-folder ${{inputs.checkpoint_folder}}
+  --input-json ${{inputs.input_json}}
+
+inputs:
+  checkpoint_folder:
+    type: uri_folder
+    path: ./checkpoints/checkpoint_000010
+    mode: download
+  input_json:
+    type: uri_file
+    path: ./init_conditions.json
+    mode: download
+environment: azureml:aml-environment@latest
+compute: azureml:medium-env
```

Comment:
I think we should make this equal to the getting started sample: ``env-medium``.